frontend/src/component/project/Project/ProjectSettings/Settings/EditProject/UpdateProject.tsx

```diff
@@ -66,7 +66,7 @@ export const UpdateProject = ({ project }: IUpdateProject) => {
     } = useProjectForm(
         id,
         project.name,
-        project.description,
+        project.description || undefined,
```

Comment:
would || null work? if you're setting it manually in the code it seems more idiomatic to be null and leave undefined for things that no one ever set explicitly